src/brouwers/kitreviews/tests/test_members.py

```diff
@@ -0,0 +1,42 @@
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+
+from brouwers.users.tests.factories import UserFactory
+from ..models import KitReview, KitReviewVote, MIN_RATING, MAX_RATING
+from .factories import KitReviewFactory, KitReviewVoteFactory
+
+
+class KitReviewTest(TestCase):
+    """
+    Test the correct behaviour of brouwers.kitreviews.models.KitReview.
+    """
+
+    def test_kit_review_rating(self):
+        kit_review1 = KitReviewFactory.create()
+        self.assertEqual(kit_review1.rating, 50)
+
+        kit_review2 = KitReviewFactory.create(rating=99)
+        self.assertEqual(kit_review2.rating, 99)
```

Comment:
This does not make much sense, your testing that factory boy and/or django work as they should. You can leave this out.